packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts

```diff
@@ -20,14 +20,7 @@ export const GET_PROCESS_DETAIL_GRAPHQL_QUERY: TypedDocumentNode<
                 lastStep
                 workflowName
                 isTask
-                form {
-                    additionalProperties
-                    definitions
-                    properties
-                    required
-                    title
-                    type
-                }
+                form
```

Comment:
Does this mean all these props are still there you just dont need to request explictly request them?

Reply:
yes and you can't request them explicitly anymore

Reply:
going to create a backend 2.1.0 release and add a changeset that this requires backend 2.1.0